lib/workos/types.rb

```diff
@@ -1,15 +1,11 @@
 # frozen_string_literal: true
-# typed: strong
```

Comment:
Had to remove this from this file to get `srb tc` to pass on `require_all`. Would the real solution be to write types for it?

https://sorbet.org/docs/error-reference#7018

Reply:
🤔 Not sure, I don't know Sorbet that well tbh